readthedocs/search/utils.py

```diff
@@ -82,27 +82,6 @@ def remove_indexed_files(model, project_slug, version_slug=None, build_id=None):
         log.exception('Unable to delete a subset of files. Continuing.')
 
 
-# TODO: Rewrite all the views using this in Class Based View,
-# and move this function to a mixin
-def get_project_list_or_404(project_slug, user, version_slug=None):
-    """
-    Return list of project and its subprojects.
-
-    It filters by Version privacy instead of Project privacy,
-    so we can support public versions on private projects.
```

Comment:
We lost this comment. Why did we remove this function in this PR? I'd really like to keep these changes small, and this PR has a number of different refactors all put together, which makes it harder to think about.

Reply:
It wasn't used in any other part, I can put back the comment about privacy in `get_all_projects`